src/guide/jbs-jdk-bug-system.md

```diff
@@ -5,52 +5,290 @@
+The most common Issue types are:
+<br />
+<table class="dictionary" summary="JBS Label Dictionary">
+  <tr style="text-align:left;"><th>Type</th><th>Covers</th></tr>
+<tr>
+    <td class="dictionary">[Bug]{.jbs-field}</td>
+    <td class="dictionary">Used when reporting a problem: crashes; hangs; failures of functionality etc.</td>
+</tr>
+<tr>
+    <td class="dictionary"><i>Improvement</i></td>
+    <td class="dictionary">When requesting an improvement in the JDK, the issue type used depends on the size/scope of the improvement:<br />
+    ([Enhancement]{.jbs-field}) - for a small improvement to existing functionality<br />
+    ([New Feature]{.jbs-field}) - not recommended to use<br />
+([JEP]{.jbs-field}) - for a proposal for a significant change or new feature which will take 4 weeks or more of work - see [JEP-1](https://openjdk.org/jeps/1)</td>
+</tr>
+<tr>
+    <td class="dictionary">[Sub-task]{.jbs-field}</td>
+    <td class="dictionary">Can be used to break up the work for an issue, such as the changes to the docs, tests etc. This is not recommended as a way to break up a large amount of code change associated with a new feature - see "Implementing a JEP or any Large Change" below</td>
+</tr>
+<tr>
+    <td class="dictionary">[Task]{.jbs-field}</td>
+    <td class="dictionary">Where something needs to happen other than a code change - a request for a new JBS version value for example</td>
+</tr>
+<tr>
+    <td class="dictionary"><i>Vulnerability</i></td>
+    <td class="dictionary">If you suspect that the issue is a vulnerability, **don't file a JBS issue**, instead send your report to [vuln-report@openjdk.org](mailto:vuln-report@openjdk.org). <br>Please do *not* report or discuss potential vulnerabilities on any open lists or other public channels. <br>See [OpenJDK Vulnerabilities](https://openjdk.org/groups/vulnerability/report) for more information.</td>
+</tr>
+</table>
 
-A few things to keep in mind when filing a new bug:
+<br>
+A few things to keep in mind when filing an issue to report a problem:
 
-* Before filing a bug, verify that there isn't already a bug filed for this issue.
-  * Search [JBS](https://bugs.openjdk.org/) for things like the name of the failing test, assert messages, the name of the source code file where a crash occurred etc.
-* If you suspect that the bug is a vulnerability, **don't file a JBS issue**. Instead send your bug report to [vuln-report@openjdk.org](mailto:vuln-report@openjdk.org).
-  * Please do *not* report or discuss potential vulnerabilities on any open lists or other public channels.
-  * See [OpenJDK Vulnerabilities](https://openjdk.org/groups/vulnerability/report) for more information.
+* Before filing, verify that there isn't an issue already filed.
```

Comment:
Made changes for your first comment.  You second is interesting - there is work happening to make it easier to add a comment to an existing bug via bugs.java.com and have added this for the case of reopening:

* if you find a similar issue that was closed as "not reproducible" then it may be appropriate to re-open that one - if you don't have direct access to JBS you can file a bug at [bugs.java.com](https://java.bugs.com) requesting that it be reopened.